lib/src/screens/homepage.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'dart:convert';
+
+Future getSpecieData(String name, BuildContext context) async {
```

Comment:
Move this logic to a separate file, which will act as a data source for our app.

Reply:
ok I will look into this @rishabh3112

Reply:
This will definately shift to a new file when cards for each animal will be created so no worries there

Reply:
Okay!
I was lookin' to implement routes using [fluro](https://github.com/theyakka/fluro). WDYT?